pallets/staking/src/lib.rs

```diff
@@ -1334,7 +1334,7 @@ pub mod pallet {
             }
 
             let post_info_weight =
-                if ledger.unlocking.is_empty() && ledger.active <= T::Currency::minimum_balance() {
+                if ledger.unlocking.is_empty() && ledger.active <= <ExistentialDeposit<T>>::get() {
```

Comment:
https://github.com/paritytech/substrate/blob/master/frame/staking/src/pallet/mod.rs#L1024
See this link, maybe no need to change